extensions/amp-bind/0.1/test/test-bind-integration.js

```diff
@@ -475,4 +487,45 @@ describe.configure().retryOnSaucelabs().run('amp-bind', function() {
       });
     });
   });
+
+  describe('amp-state', () => {
+    it('should not loop infinitely if updates change its src binding', () => {
+      // Changes amp-state's src to ...amp-state-src2
```

Comment:
d'oh, those are the values I originally had for `src` that I changed for readability reasons. Fixed.